data-raw/family.R

```diff
@@ -0,0 +1,15 @@
+library(dplyr)
+
+family <- tribble(
+  ~family,  ~dob_child1,  ~dob_child2, ~name_child1, ~name_child2,
+        1, "1998-11-26", "2000-01-29",      "Susan",       "Jose",
+        2, "1996-06-22",           NA,       "Mark",           NA,
+        3, "2002-07-11", "2004-04-05",        "Sam",       "Seth",
+        4, "2004-10-10", "2009-08-27",      "Craig",       "Khai",
+        5, "2000-12-05", "2005-02-28",     "Parker",     "Gracie",
+)
+family <- family |>
+  mutate(across(starts_with("dob"), parse_date))
+family
+
+usethis::use_data(family, overwrite = TRUE)
```

Comment:
For some reason if I try and run this code then git tells me that `family.rda` has changed. Not sure if that is worrying or not.

Even if I load up tidyr and immediately resave the `family` dataset from tidyr, I get a diff. So I guess it has something to do with `use_data()`

Reply:
This is true, so it makes me feel a little better, still not sure what is happening when I save it though

``` r
library(tidyverse)
#> Warning: package 'readr' was built under R version 4.1.2
#> Warning: package 'dplyr' was built under R version 4.1.2

family2 <- tribble(
  ~family,  ~dob_child1,  ~dob_child2, ~name_child1, ~name_child2,
  1, "1998-11-26", "2000-01-29",      "Susan",       "Jose",
  2, "1996-06-22",           NA,       "Mark",           NA,
  3, "2002-07-11", "2004-04-05",        "Sam",       "Seth",
  4, "2004-10-10", "2009-08-27",      "Craig",       "Khai",
  5, "2000-12-05", "2005-02-28",     "Parker",     "Gracie",
)
family2 <- family2 |>
  mutate(across(starts_with("dob"), parse_date))

identical(family, family2)
#> [1] TRUE
```

<sup>Created on 2022-03-09 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Reply:
Yeah, I noticed that too. I have a vague recollection that maybe rda includes a time stamp?

Reply:
At the very least, it adds the R version in there, which likely explains our differences
https://github.com/wch/r-source/blob/6b22b60126646714e0f25143ac679240be251dbe/src/main/serialize.c#L1397